mdoc/mdoc.Test/FormatterTests.cs

```diff
@@ -207,6 +207,15 @@ public void IL_RefAndOut ()
                 "~SomeGenericClass ();",
                 "Finalize");
 
+        [Test]
+        public void CSharpReadonlyRef()
+        {
+            var member = GetMethod(typeof(ReadonlyRefClass), m => m.Name == "ReadonlyRef");
+            var formatter = new CSharpFullMemberFormatter();
+            var sig = formatter.GetDeclaration(member);
+            Assert.AreEqual("public ref readonly int ReadonlyRef ();", sig);
```

Comment:
This unit test is great … but I'd like to see a few more combinations of unit tests, such as a method that returns only `ref` (I _believe_ you can't have only a `readonly` returns, right?). But in any case, unit tests for every variation would be ideal.

Reply:
good catch! Added another case for `ref` return only, and found a bug in my change 😉 . Updated.